### PR TITLE
Spadimis/permissions reference

### PR DESCRIPTION
**Pull request by Sureshpadimi88**

### CODE REVIEW

File: concepts/permissions-reference.md [hunk 1]
```diff
@@ -876,11 +876,11 @@ For more complex scenarios involving multiple permissions, see [Permission scena
...
+| _EduAssignments.ReadWriteBasic.All_           | Create,read,update and delete all class assignments without grades              |Allows the app to create,read, update and delete all class assignments without grades for all users without a signed-in user                                                                   | Yes                    |
+| _EduAssignments.ReadBasic.All_     | Read all class assignments without grades      | Allows the app to read all class assignments without grades for all users without a signed-in user                                                                                                    | Yes                    |
+| _EduRoster.ReadBasic.All_            | Read a limited subset of the organization's roster. | Allows the app to read a limited subset of both the structure of schools and classes in an organization's roster and education-specific information about all users.          | Yes                    |
```
Comment (cristobal-buenrostro) [inline concepts/permissions-reference.md:883]: something is wrong, it seems like you reverted the changes. 
Please check and update accordingly.

**Review by cristobal-buenrostro — CHANGES_REQUESTED** [review]
@Sureshpadimi88 , please remove all the files from **api-reference** folder.
the only file needed in this PR is **concepts/permissions-reference.**

File: concepts/permissions-reference.md (hunk 1 above)
Comment (Sureshpadimi88) [reply concepts/permissions-reference.md:883]: 151465 
update with these new descriptions

Image

Comment (Sureshpadimi88) [reply concepts/permissions-reference.md]: it's done.

**Pull request closed without merging**